c7n/utils.py

```diff
@@ -366,14 +366,17 @@ def get_retry(codes=(), max_attempts=8, min_delay=1, log_retries=False):
     """
     max_delay = max(min_delay, 2) ** max_attempts
 
-    def _retry(func, *args, **kw):
+    def _retry(func, ignore_err_codes=None, *args, **kw):
```

Comment:
I think you need to put `ignore_err_codes` after `*args` since you'd get an exception if you sent positional arguments after a keyword argument in the call.

Reply:
noted and thanks, fwiw, I'm not sure if we do have any positional args, the boto client only takes keywords args on api calls.